be/CMakeLists.txt

```diff
@@ -69,6 +69,9 @@ endif()
 # enable glog custom prefix
 add_definitions(-DGLOG_CUSTOM_PREFIX_SUPPORT)
 
+# enable simdjson check eof
+add_definitions(-DSIMDJSON_CHECK_EOF)
```

Comment:
The official comment in simdjson does not suggest turn it on.

`option(SIMDJSON_CHECK_EOF "Check for the end of the input buffer. The setting is unnecessary since we require padding of the inputs. You should expect tests to fail with this option turned on." OFF)`

Reply:
if we do not add this, we should check in our logic for input string buffer which has error ending